### ISSUE TITLE
NTR: L-2-hydroxyglutaric acid high in CSF (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8343]
New term request L-2-hydroxyglutaric acid high in CSF (elthat_130528183340): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Is the L appropriate here? Most sources do not specify chiralty
https://en.wikipedia.org/wiki/2-Hydroxyglutaric_aciduria

**Comment by MickeySegal** [comment]
The chirality is important.  D-2-hydroxyglutaric acid is elevated in some conditions, e.g., https://omim.org/entry/600721.  Both D and L are elevated in https://omim.org/entry/615182.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000509
- label: Elevated CSF L-2-hydroxyglutaric acid concentration
- parents: [Abnormal CSF carboxylic acid concentration (HP:0500183)](https://hpo.jax.org/app/browse/term/HP:0500183)
- pmid: 
- def: The concentration of L-2-hydroxyglutaric acid in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8343
- synonyms:
